Misc/NEWS.d/next/Library/2024-01-12-09-35-07.gh-issue-112202.t_0V1m.rst

```diff
@@ -1 +1 @@
-Ensure that a :func:`asyncio.Condition.notify` call does not get lost if the notified ``Task`` is simultaneously cancelled or encounters any other error.
+Ensure that a :func:`asyncio.Condition.notify` call does not get lost if the awoken ``Task`` is simultaneously cancelled or encounters any other error.
```

Comment:
```suggestion
Ensure that a :func:`asyncio.Condition.notify` call does not get lost if the awakened ``Task`` is simultaneously cancelled or encounters any other error.
```

Reply:
Thanks, this is territory where my brain hasn't formed the correct nodes.